exla/test/exla/defn_expr_test.exs

```diff
@@ -2568,6 +2568,26 @@ defmodule EXLA.DefnExprTest do
     end
   end
 
+  describe "gather" do
+    defn(gather(t, idx), do: Nx.gather(t, idx))
```

Comment:
```suggestion
    defn gather(t, idx), do: Nx.gather(t, idx)
```

Are the parens needed?

Reply:
Nope, probably formatted within the editor and it didn't pick up the formatter config :)